SSRSUsageQueries/SSRSUsageQueries/Summary.sql

```diff
@@ -0,0 +1,18 @@
+use SSRSUsageQueries;
```

Comment:
Why is this the database name used? You should take this part out since you don't know in advance what's the reporting database name...